packages/oc-template-handlebars-compiler/test/compile.test.js

```diff
@@ -19,20 +37,115 @@ test('Should correctly compile the oc component', done => {
       version: '1.0.0'
     },
     componentPath,
-    publishPath
+    publishPath: path.join(componentPath, '_compile-tests-package')
   };
-  fs.ensureDirSync(publishPath);
-  compile(options, (err, res) => {
+
+  init(options, (err, { result, files }) => {
```

Comment:
As you are doing multiple scenario tests, could be nice perhaps to create some scenarios = [, , , ] and test through each of them once